static/js/hashchange.js

```diff
@@ -136,7 +138,23 @@ function do_hashchange_normal(from_reload) {
         case "#narrow": {
             maybe_hide_recent_topics();
             ui_util.change_tab_to("#message_feed_container");
-            const operators = hash_util.parse_narrow(hash);
+            let operators = undefined;
+            try {
+                // This fails for URLs containing
+                // foo.foo or foo%foo due to our fault in special handling
+                // of such characters when encoding. This can also,
+                // fail independent of our fault, so just tell the user
+                // that the URL is invalid.
+                // TODO: Show possible valid URLs to the user.
```

Comment:
Does it makes sense to duplicate this comment?

Reply:
It does not, that was a rebasing error. I've split the comment in a way I think makes sense.